ansible/provision.yml

```diff
@@ -27,6 +27,14 @@
     - common
     - mongodb
 
+- name: Message queue
+  user: ubuntu
+  hosts: tag_type_mq:mq
```

Comment:
back again to this line.

why do we need `tag_type_mq:mq`.  Isn't `mq` defined as all the instances in `mq_set`, which is itself already intersected with `tag_type_mq`?  Partially I'm asking to see if my understanding is correct.  This may be clearer as `mq` if what I'm saying holds.